adobe_vipm/flows/fulfillment/shared.py

```diff
@@ -992,6 +1006,22 @@ def __call__(self, client, context, next_step):  # noqa: C901
             return
         next_step(client, context)
 
+    @staticmethod
+    def _get_flex_discounts_params(client: MPTClient, adobe_order: dict, order_id: str):  # noqa: WPS602
```

Comment:
pls, remove the static decorator according to the ignored rule.

Also the return type is missed here